### DISCUSSION TITLE
Feed Grouping?

### DISCUSSION THREAD

**Discussion by juliamayrhauser** [dailydotdev/daily#1081] [General]
Hi, 

I had an idea for a cool feature. Not sure if this has been done yet though, at least I didn't find it: It would be cool if you could group your feed, so you have a general feed that displays articles for the tags you follow and then you can have different sub-feeds for specific tags only. For example, I'm a fullstack dev, but currently I'm very interested in UX and trying to stay up to date with UX trends. I don't wanna unfollow all my other tags, but I also want more focus on UX without having to apply a filter each time. 

What do you think? Does that make sense? Would anyone else have a use for this?

**Comment by idoshamun** [comment]
Hi @juliamayrhauser,

This idea makes perfect sense. We already have it in the backlog for a while but not prioritized at the moment unfortunately.
Anyway, let's keep the discussion here: #1093
